Style.css

```diff
@@ -169,6 +173,103 @@ h1 {
   color: #172b4d;
 }
 
+/* pop-up menu */
+#showDetails {
+  display: none;
+}
+
+#showDetails.activate {
+  position: relative;
+  top: -6vh;
+  z-index: 1;
+  display: flex;
```

Comment:
- [ ] Please fix the below issues for desktop screens and make sure your design matches with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=JisnhgjX2CTJQVt3-0) design.
  - Please add some top margin to the pop-up(model).
  - Please remove the background color from the close(X) button.
  - Please make sure that the project title, tags, and images start on the same vertical line.
  - Please make sure the project description section and (tags, and buttons) sections start on the same horizontal line.
  - Please make sure that the project tags and button section are aligned on the right side of the pop-up(model).
  - Please add the below interactions to the pop-up(model) buttons.
  ![image](https://user-images.githubusercontent.com/83803414/208910865-14f15413-14e0-442a-88a5-c65a93401903.png)



### Your design

![image](https://user-images.githubusercontent.com/83803414/208912405-ea7cd638-a9e8-4d02-a528-b04e628b0a5b.png)

![image](https://user-images.githubusercontent.com/83803414/208912315-5d4f6321-b0cf-4fc8-aac4-f567ed8c974c.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/208912815-5c90e2c0-fc39-41cd-a417-937878540ff2.png)

![image](https://user-images.githubusercontent.com/83803414/208912755-cc1fb260-1bcb-4804-88a2-5268f8416d41.png)